extensions-core/kinesis-indexing-service/src/main/java/org/apache/druid/indexing/kinesis/KinesisIndexTaskRunner.java

```diff
@@ -125,28 +125,42 @@ protected void possiblyResetDataSourceMetadata(
   {
     if (!task.getTuningConfig().isSkipSequenceNumberAvailabilityCheck()) {
       final ConcurrentMap<String, String> currOffsets = getCurrentOffsets();
+      final Map<StreamPartition<String>, String> shardResetMap = new HashMap<>();
```

Comment:
nit: for clarity, maybe call this map `partitionToSequenceResetMap` since we refer to the keys and values of the map a few times?

Reply:
Done